### ISSUE TITLE
Project Already Exists, But It Doesn't

### ISSUE THREAD

**Issue by zachdeibert** [rtfd/readthedocs.org#4027]
## Details

* Read the Docs project URL: zenith.readthedocs.io
* Build URL (if applicable):
* Read the Docs username (if applicable): zachdeibert

## Expected Result

When creating a project with a name that does not exist (reported by the [API](http://readthedocs.org/api/v1/project/?slug=zenith)), the project should be created.

## Actual Result

Project creation fails with "Invalid project name, a project already exists with that name."

**Comment by stsewd** [comment]
The project exists https://readthedocs.org/projects/zenith/, it doesn't have any successful build, so maybe you want to check https://docs.readthedocs.io/en/latest/abandoned-projects.html if you want the name

**Comment by zachdeibert** [comment]
How would I go about starting the process for getting the name?

**Comment by stsewd** [comment]
@zachdeibert seems that this is the user https://github.com/seacen. @humitos can help you with the process.

**Comment by humitos** [comment]
Maybe the project doesn't appear in the API response because its privacy level is "protected"

**Comment by zachdeibert** [comment]
Yeah, I just created a protected project and it only shows up in the API responses if I send my session token with it.  I didn't realize that protected could keep it out of that list.
